src/main/scala/io/iohk/ethereum/network/handshaker/EtcHelloExchangeState.scala

```diff
@@ -24,10 +26,14 @@ case class EtcHelloExchangeState(handshakerConfiguration: EtcHandshakerConfigura
 
   override def applyResponseMessage: PartialFunction[Message, HandshakerState[PeerInfo]] = { case hello: Hello =>
     log.debug("Protocol handshake finished with peer ({})", hello)
-    if (hello.capabilities.contains(Capability("eth", Versions.PV63.toByte)))
-      EtcNodeStatusExchangeState(handshakerConfiguration)
+    if (handshakerConfiguration.protocolVersion == Versions.PV64 && hello.capabilities.contains(Eth64Capability))
```

Comment:
Should we add a comment on our `protocol-version` from the application.conf on how it impacts on this, the problems with eth64 and how it should be used?